Header header
geometry_msgs/Vector3 b1
geometry_msgs/Vector3 xd
geometry_msgs/Vector3 xd_dot
geometry_msgs/Vector3 xd_ddot
